go.mod

```diff
@@ -0,0 +1,20 @@
+module github.com/observatorium/obsctl
+
+go 1.17
+
+require (
+	github.com/bwplotka/mdox v0.9.0
+	github.com/efficientgo/tools/extkingpin v0.0.0-20210609125236-d73259166f20
+	github.com/go-kit/log v0.2.0
+	github.com/oklog/run v1.1.0
+	github.com/pkg/errors v0.9.1
+	gopkg.in/alecthomas/kingpin.v2 v2.2.6
```

Comment:
I am not sure if we want to use `kingpin` for a new project. It's in maintenance only mode for quite a while and the last commit was in 2017.

I'll try looking around to see if I can find a good alternative.

Reply:
Looks like [cobra](https://github.com/spf13/cobra) is the most widely used one right now. Another one is https://github.com/urfave/cli

Reply:
Ack! Yeah, the release I was using here is old, plus kind of wanted to have some "mutual exclusiveness" among certain flags as well which kingpin doesn't support yet. Will try to change it to cobra. 🙂

Reply:
nbd, that is what is used in k8s CLIs too :nail_care:

Reply:
Changed this to cobra [here](https://github.com/observatorium/obsctl/pull/1/commits/161bd0a8fddee3d6fc4e8174087b7acf1d246f32). There are a few caveats to this, like not being able to explicitly mentions args for a command, but it works well for larger projects, so likely will find workarounds! 🙂